src/libs/Localize/LocaleListener/index.desktop.js

```diff
@@ -0,0 +1,13 @@
+import ELECTRON_EVENTS from '../../../../desktop/ELECTRON_EVENTS';
+import BaseLocaleListener from './BaseLocaleListener';
+
+const listenForLocaleChanges = () => {
```

Comment:
Two comments:

1. This function should take a callback that's executed when the locale changes
1. NAB but I think it might be simpler for this function to use the same name as the `BaseLocaleListener`:

```suggestion
const connect = (callback) => {
    BaseLocaleListener.connect((val) => {
    	callback(val);

    	// Send the updated locale to the Electron main process
    	window.electron.send(ELECTRON_EVENTS.LOCALE_UPDATED, val);
    });
};
```

Reply:
Hmmm, I was just saying that, as a consumer of this library, I would expect to be able to pass a callback function to `LocaleListener.connect` like so:

```js
LocaleListener.connect(updatedLocale => // do something with updated locale);
```

NAB if it's not needed right now